draft-ietf-rtcweb-jsep.xml

```diff
@@ -5680,6 +5582,167 @@ a=rtcp-fb:100 nack pli
         <xref target="I-D.ietf-mmusic-dtls-sdp" />Section 4</c>
       </texttable>
     </section>
+
+    <section title="Appendix B" anchor="sec.appendix-b">
+      <t>The following text is meant to completely replace section
+      "Associating RTP/RTCP Streams With Correct SDP Media
+      Description" of mmusic-sdp-bundle-negotiation. </t>
+      
+        <t>
+          As described in RFC3550, RTP/RTCP packets
+          are associated with RTP streams as defined in RFC7656. Each RTP stream is identified by an SSRC value, and each
```

Comment:
You should be consistent about whether you are going to use cross-refs or just RFC literals here. I would prefer xrefs.